pyproject.toml

```diff
@@ -28,7 +28,6 @@ pynufft-cpu = ["pynufft"]
 pynufft-gpu = ["pynufft"]
 
 io = ["pymapvbvd"]
-smaps = ["scikit-image"]
```

Comment:
Why though?

Reply:
smaps estimation is part of the extra module, which already has scikit-image as dependency, so let's just make things simple and reduce the number of possibility to install.